src/mailadm/cmdline.py

```diff
@@ -125,7 +125,7 @@ def setup_bot(ctx, email, password, show_ffi):
         ac.set_config("addr", email)
     if password:
         ac.set_config("mail_pw", password)
-    ac.set_config("mvbox_move", "0")
+    ac.set_config("mvbox_move", "1")
```

Comment:
Mailcow has a default Sieve rule which moves mails to the DeltaChat folder - they can be overlooked by the bot in this case. I turned on this option so the bot watches the DeltaChat folder and doesn't miss messages. It also moves the messages to the DeltaChat folder, which doesn't really hurt, as long as the bot does not need to save bandwidth (it typically is running on the same server as mailcow, so nope, shouldn't be a problem).